code/datums/wires/syndicatebomb.dm

```diff
@@ -84,8 +100,11 @@
 				B.update_icon()
 
 /datum/wires/syndicatebomb/proc/tell_admins(obj/machinery/syndicatebomb/B)
-	if(istype(B, /obj/machinery/syndicatebomb/training))
+	if(!is_real_bomb(B))
 		return
 	var/turf/T = get_turf(B)
-	log_game("\A [B] was detonated via boom wire at [AREACOORD(T)].")
-	message_admins("A [B.name] was detonated via boom wire at [ADMIN_VERBOSEJMP(T)].")
+	log_game("\A [B] was detonated via boom wire at [AREACOORD(T)][B.fingerprintslast ? "by [B.fingerprintslast]" : ""]")
+	message_admins("A [B.name] was detonated via boom wire at [ADMIN_VERBOSEJMP(T)][B.fingerprintslast ? "by [B.fingerprintslast]" : ""].")
```

Comment:
Would this account for remote signalling device and someone else touching it afterwards? Or someone installing a remote signalling device and having someone else pulse it? Instead of saying by [x] it should probably say Fingerprint Last: [x]